docs/rendering/modelextensions/visibility.md

```diff
@@ -0,0 +1,12 @@
+Part Visibility
+===============
+
+Adding the `visibility` entry at the top level of a model JSON allows control over the visibility of different parts of the model to decide whether they should be baked into the final [`BakedModel`][bakedmodel]. The definition of a "part" is dependent on the model loader loading this model. Out of the model loaders provided by Forge only the [composite model loader][composite] and the [OBJ model loader][obj] make use of this functionality.
+
+The visibility entries are specified as `"part name": boolean` entries. The check for the visibility of a given part traverses the full dependency chain until it finds an entry or reaches the end of the chain such that if a model doesn't specify the visibility of a part its parent will be interrogated for the visibility of that part. If no model in the dependency chain specifies a visibility for a part the visibility will default to true.
```

Comment:
Is this just saying it finds the visibility of the part by checking whether its specified and recursively checks the parent until either it finds an instance or is not specified, so it defaults to true? If so, I would say the language is not modder-friendly and probably should be simplified or given an example.

Reply:
Simplified the explanation, I hope it makes more sense this way